src/Enterspeed.Source.UmbracoCms.V10/Services/DataProperties/DefaultConverters/DefaultDateTimePropertyValueConverter.cs

```diff
@@ -16,7 +15,7 @@ public bool IsConverter(IPublishedPropertyType propertyType)
         public virtual IEnterspeedProperty Convert(IPublishedProperty property, string culture)
         {
             var date = property.GetValue<DateTime>(culture);
-            return new StringEnterspeedProperty(property.Alias, date.ToString(CultureInfo.InvariantCulture));
+            return new StringEnterspeedProperty(property.Alias, date.ToString("yyyy-MM-ddTHH:mm:ss"));
```

Comment:
Now that we are addressing it. I can see this format being used across the connectors. Should we add it as a const?

Currently, we do not have any shared projects across the projects, but for each individual connector, this is repeated 4 times.

Just a thought :) 

![image](https://user-images.githubusercontent.com/61312252/194727188-67cfc343-f097-40c4-af3e-931df2053895.png)

Reply:
Yes, I agree. I also thought about if we should create a core project (maybe just one for the v9 and v10), since most of the logic for these two projects are identical, and especially when v11 and v12 is coming soon as well.

What do you think about that?

Reply:
For now I can just create a constant of an extension method on DateTime and then, if we like, I can create a story in shortcut to create a core project so we don't clutter this story.

Reply:
This has actually been something that we have discussed multiple times. I fully agree :)